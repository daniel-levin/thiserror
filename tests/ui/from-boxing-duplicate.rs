use thiserror::Error;

#[derive(Debug, Error)]
pub struct S {
    #[from(boxing)]
    a: std::io::Error,

    #[from(boxing)]
    b: std::io::Error,
}

fn main() {}
