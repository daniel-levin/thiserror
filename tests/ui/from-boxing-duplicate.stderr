error: duplicate #[from] attribute
 --> tests/ui/from-boxing-duplicate.rs:8:5
  |
8 |     #[from(boxing)]
  |     ^^^^^^^^^^^^^^^
